examples/react-native/src/features/Authenticator/Demo/Demo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
+import { Amplify } from 'aws-amplify';
+
+import { Button } from '../../../ui';
```

Comment:
For the storybook stories we've been importing from `@aws-amplify/ui-react-native/dist/primitives`

Reply:
Well you could be importing from the `ui` folder 😏

Reply:
That was kind of my intention when i added it initially 😅 Since we haven't having been using this, cool if i just clean up and remove the `ui` folder in a follow up?